src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -247,6 +247,33 @@ object StringUtils {
         } ?: text
     }
 
+    fun String.wrapText(maxLength: Int): List<String> {
+        val words = this.split(" ")
+        val wrappedLines = mutableListOf<String>()
+        var currentLine = ""
+
+        for (word in words) {
+            when {
+                currentLine.length + word.length + 1 > maxLength -> {
+                    wrappedLines.add(currentLine)
+                    currentLine = word
+                }
+
+                currentLine.isEmpty() -> currentLine = word
+                else -> currentLine += " $word"
+            }
+
+            if (word.length > maxLength) {
+                wrappedLines.addAll(word.chunked(maxLength))
+                currentLine = ""
+            }
+        }
+
+        if (currentLine.isNotEmpty()) wrappedLines.add(currentLine)
+
+        return wrappedLines
+    }
```

Comment:
Ehm, why do you need this. Minecraft's default font is not a mono meaning this would not do what you would expect. Plus there is already a function from mojang present. (Look at Renderable.wrappedString for details).

Reply:
If I dont use this, the full text of any long description (like Diana event or Cole event) will be in one/two lines. 
But thanks, I'll look into that soon hopefully